c2rust-analyze/src/rewrite/mod.rs

```diff
@@ -74,6 +76,10 @@ pub enum Rewrite<S = Span> {
     TySlice(Box<Rewrite>),
     /// `Foo<T1, T2>`
     TyCtor(String, Vec<Rewrite>),
+
+    // `static` builders
+    /// `static` mutability (`static` <-> `static mut`)
+    StaticMut(Mutability, S),
```

Comment:
It's unfortunate that this requires such a weird and specialized rewrite.  I was going to suggest adding a `Rewrite::Delete` variant that simply deletes the text in the span it's applied to, but looking at `hir::Item`, getting the exact span of the `mut` token looks like it would be a real pain.